src/client/theme-default/styles/components/vp-doc.css

```diff
@@ -426,9 +426,6 @@
   /*rtl:ignore*/
   right: 12px;
   z-index: 3;
```

Comment:
```suggestion
  z-index: 3;
  display: block;
```

Reply:
`button` has already set `position:absolute`, at this time the computed value of the `display` property becomes 'block', so I think it can be removed here.

Reply:
> button has already set position:absolute, at this time the computed value of the display property becomes 'block'

How?? On Chrome it's using the user agent stylesheet that has display: inline-block